test/EFCore.PG.FunctionalTests/Query/ComplexNavigationsWeakQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.EntityFrameworkCore.Query;
+using Xunit.Abstractions;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    public class ComplexNavigationsWeakQueryNpgsqlTest : ComplexNavigationsWeakQueryTestBase<ComplexNavigationsWeakQueryNpgsqlFixture>
+    {
+        // ReSharper disable once UnusedParameter.Local
+        public ComplexNavigationsWeakQueryNpgsqlTest(
+            ComplexNavigationsWeakQueryNpgsqlFixture fixture, ITestOutputHelper testOutputHelper)
```

Comment:
Why line wrap here?

Reply:
Just because the class name is big and so the long becomes very long. Changed to a single line.